src/pages/portfolio/index.tsx

```diff
@@ -99,28 +108,57 @@ const useStyles = makeStyles((theme) => ({
   },
   title: {
     marginBottom: theme.spacing(4),
+    width: '100%',
+    zIndex: 1,
 
     [theme.breakpoints.up('sm')]: {
+      position: 'absolute',
       marginBottom: theme.spacing(3),
     },
 
     [theme.breakpoints.up('lg')]: {
+      position: 'absolute',
       margin: theme.spacing(0, 2, 4, 2),
     },
   },
+  navbarTitles: {
+    position: 'relative',
+    zIndex: 2,
+    margin: theme.spacing(-1.2, 2, 4, 2),
+  },
 }));
 
 const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
-  const projectData = data.portfolioPage.frontmatter;
   const classes = useStyles();
-  const { componentType, isDesktop } = useComponentType();
+  const defaultLabel = 'All';
+  const projectData = data.portfolioPage.frontmatter;
+  const projectLabels = projectData.projects;
+  const { componentType, isDesktop, isMobile } = useComponentType();
   const developerProfile = useDeveloperProfile();
   const [selectedProject, setSelectedProject] = useState(-1);
+  const [navbarTitle, setNavbarTitle] = useState(0);
+  const projectLabelsTitles = [defaultLabel, ...new Set(projectLabels.map((project) => project.projectLabel))];
```

Comment:
If you rename as suggested above, please do like so 😄 
```suggestion
  const projectLabels = [defaultLabel, ...new Set(projects.map((project) => project.projectLabel))];
```